src/c_wrapper.jl

```diff
@@ -86,6 +100,11 @@ end
 # Take Ref{}s because SCS might modify the structs
 for (T, lib) in zip([SCS.Direct, SCS.Indirect], [SCS.direct, SCS.indirect])
     @eval begin
+
+        function SCS_set_default_settings!(::Type{$T}, data::Ref{SCSData})
```

Comment:
Naming nit: The name implies modification, so JuMP style recommends dropping the `!` from the name (http://www.juliaopt.org/JuMP.jl/latest/style.html#Use-of-!-1). This is also consistent with `SCS_init`, `SCS_solve`, and `SCS_finish`.